final/RocketReserver/TripsBadgeView.swift

```diff
@@ -0,0 +1,39 @@
+import SwiftUI
+
+struct TripBadgeView : View {
+
+    private let size = 16.0
+    private let x = 20.0
+    private let y = 0.0
+
+    @StateObject private var viewModel = TripsBadgeViewModel()
+
+    var body: some View {
+        ZStack {
+            Capsule()
+            .fill(.red)
+            .frame(width: size * widthMultplier(), height: size, alignment: .topTrailing)
+            .position(x: x, y: y)
+
+            Text("\(viewModel.count)")
+            .foregroundColor(.white)
+            .font(Font.caption)
+            .position(x: x, y: y)
+        }
+        .task {
+            viewModel.loadUserTrips()
+        }
+    }
+
+    func widthMultplier() -> Double {
```

Comment:
I feel like there should be a way to just use the label's size here, but I don't know enough SwiftUI. Feels like there should be an equivalent to `UILabel.sizeToFit()`.

Reply:
Hahaha Hi Ellen! Thanks for the tip!